client/landing/jetpack-cloud/components/masterbar/index.jsx

```diff
@@ -27,7 +27,7 @@ const JetpackCloudMasterBar = () => {
 	const headerTitle = useSelector( getDocumentHeadTitle );
 	const currentRoute = useSelector( getCurrentRoute );
 	const isNarrow = useBreakpoint( '<660px' );
-	const isExteriorPage = /^\/(?:backups|scan)\/[^/]*$/.test( currentRoute );
+	const isExteriorPage = /^\/(?:backup|scan)\/[^/]*$/.test( currentRoute );
```

Comment:
This still seems to recognize the "Activity log" (Backup) and "History" (Scan) pages as not-external pages. Is this logic correct?

cc @cbauerman, since you seem to have been the original author of this bit.

Reply:
yes, that is the logic. The rational form design is that once you have navigate to an interior page the branding can collapse.